about_us.html

```diff
@@ -185,14 +184,18 @@
                     Coeur d'Alene, ID 83814
                 </div>
             </div>
-
+            <div class="footer-bottom-row">
+                <a href="/privacy-policy.html" class="nav-item">Privacy Policy</a>
+            </div>
             <div class="blue-stripe-lower"></div>
         </section>
     </v-app>
 </div>
 </body>
 </html>
 
+<script src="./assets/js/cookie-consent.js" type="text/javascript"></script>
```

Comment:
do we need this import repeated so many times? may be so..

Reply:
Yeah, unless I'm missing something but we need to have that banner on every page. If there's a better way to do this let me know.

Reply:
Yeah that's what I was afraid of. Would be neat if we had a hook/template of some kind so we didn't have to do this.. oh well, works for now!